lib/pbench/test/unit/server/test_indexer.py

```diff
@@ -22,12 +22,33 @@ def test_not_found():
 
     @staticmethod
     def test_fallbacks():
-        keyvals = {"name": "mybm"}
+        templ = "%benchmark_name%_UID"
+        res = ResultData.expand_uid_template(templ, dict(foo=1, bar=2))
+        assert res == templ
+
+        keyvals = {"benchmark_name": "mybm"}
         templ = "%benchmark_name%_UID"
         res = ResultData.expand_uid_template(templ, keyvals)
         assert res == "mybm_UID"
```

Comment:
I think this case is redundant, given `test_found()`, but I s'pose it doesn't hurt.